Executive draft jan 12 2024 (#2520)
makerdao/community: governance/votes/Executive vote - January 12, 2024.md

---

> title: Template - [Executive Vote] SparkLend Changes, Compensation Distribution, Halting Legacy Streams, Offboarded Delegate Buffer Payments, and Update HVBank Docs - January 12, 2024

Reviewer: doc is lower-case and singular

```suggestion
title: Template - [Executive Vote] SparkLend Changes, Compensation Distribution, Halting Legacy Streams, Offboarded Delegate Buffer Payments, and Update HVBank doc - January 12, 2024
```

> summary: Increase the SparkLend D3M Debt Ceiling, execute SpaarkLend Prozy Spell, November and December Aligned Delegate compensation, MKR payouts for SES and BA Labs, Launch Project funding, yank DAI streams for DECO and SES, Offboarded AD Buffer MKR distribution to Navigator and PALC, and Update the HVBank (RWA009-A) doc parameter. 

Reviewer: spelling and capitalization

```suggestion
summary: Increase the SparkLend D3M Debt Ceiling, execute SparkLend Proxy Spell, November and December Aligned Delegate compensation, MKR payouts for SES and BA Labs, Launch Project funding, yank DAI streams for DECO and SES, offboarded AD Buffer MKR distribution to Navigator and PALC, and update the HVBank (RWA009-A) doc parameter. 
```

> date: 2023-01-12T00:00:00.000Z

Reviewer: 🙃 

```suggestion
date: 2024-01-12T00:00:00.000Z
```

address: "$spell_address"

---

> # [Executive Proposal] SparkLend Changes, Compensation Distribution, Halting Legacy Streams, Offboarded Delegate Buffer Payments, and Update HVBank Docs - January 12, 2024

Reviewer: ```suggestion
# [Executive Proposal] SparkLend Changes, Compensation Distribution, Halting Legacy Streams, Offboarded Delegate Buffer Payments, and Update HVBank doc - January 12, 2024
```

The Governance Facilitators, Sidestream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

> - Increase the SparkLend [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 400 million from 800 million to **1.2 billion Dai**.

Reviewer: "DAI" should be capitalized when it is used as units

```suggestion
- Increase the SparkLend [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 400 million from 800 million to **1.2 billion DAI**.
```

- Execute the SparkLend Proxy Spell at $address for updates and technical changes detailed [here](https://forum.makerdao.com/t/spark-spell-proposed-changes/23298).
- November and December Aligned Delegate Compensation will be distributed to 10 delegates.

> - BA Labs and SES will recieve **175.00** and **508.55 MKR** respectively.
> - The Launch Project will recieve **4.5 million DAI** and **820.00 MKR**.

Reviewer: spelling

```suggestion
- BA Labs and SES will receive **175.00** and **508.55 MKR** respectively.
- The Launch Project will receive **4.5 million DAI** and **820.00 MKR**.
```

> - The DAI [Token Streams](https://manual.makerdao.com/module-index/module-token-streaming#trade-offs) for DECO and SES will be halted (`yank`).

Reviewer: "Dai" should not be capitalized when it is a noun.

```suggestion
- The Dai [Token Streams](https://manual.makerdao.com/module-index/module-token-streaming#trade-offs) for DECO and SES will be halted (`yank`).
```

> - The DAI [Token Streams](https://manual.makerdao.com/module-index/module-token-streaming#trade-offs) for DECO and SES will be halted (`yank`).

Reviewer: Is linking directly to the trade-off section intentional?

> - Two Derecognized ALigned Delegates will receive their remaining surplus buffer, with Navigator receiving **20.84 MKR** and PALC **6.95 MKR**.

Reviewer: not surplus buffer

```suggestion
- Two Derecognized ALigned Delegates will receive their remaining AD buffer, with Navigator receiving **20.84 MKR** and PALC **6.95 MKR**.
```

- The HVBank (RWA009-A) [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter will be updated to to **QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT**.

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

> ### Increase the SparkLend D3M

Reviewer: ```suggestion
### Increase the SparkLend D3M Maximum Debt Ceiling
```

> ### Increase the SparkLend D3M

Reviewer: add placeholder for the poll link

**Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/spark-spell-proposed-changes/23298/5)  
**Proposal**: [Forum Post](https://forum.makerdao.com/t/spark-spell-proposed-changes/23298)

> If this executive proposal passes, the SparkLend [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased by 400 million from 800 million to **1.2 billion Dai**.

Reviewer: ```suggestion
If this executive proposal passes, the SparkLend [D3M](https://manual.makerdao.com/module-index/module-dai-direct-deposit) [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased by 400 million from 800 million to **1.2 billion DAI**.
```

> ###  Execute the SparkLend Proxy Spell

Reviewer: Consider adding a placeholder for the poll links, and possibly adding a bit more detail?

Author: We will discuss a standard practice for Spark Proxy Spells, given the circumstances this will suffice for now.

> ###  Execute the SparkLend Proxy Spell

Reviewer: Match doc order to sheet order where possible so code and copy are aligned

| Cloaky       | 41.67        | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) |
| TRUE NAME    | 41.67        | [0x612F7924c367575a0Edf21333D96b15F1B345A5d](https://etherscan.io/address/0x612F7924c367575a0Edf21333D96b15F1B345A5d) |
| BLUE         | 39.20        | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
| PBG          | 13.89        | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) |
| UPMaker      | 13.89        | [0xbB819DF169670DC71A16F58F55956FE642cc6BcD](https://etherscan.io/address/0xbB819DF169670DC71A16F58F55956FE642cc6BcD) |
| vigilant     | 13.89        | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) |
| JAG          | 12.95        | [0x58D1ec57E4294E4fe650D1CB12b96AE34349556f](https://etherscan.io/address/0x58D1ec57E4294E4fe650D1CB12b96AE34349556f) |

> | WBC          | 11.28        | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) |
> 
> 
> 
> ### Legacy Agreement MKR Distributions

Reviewer: ```suggestion
| WBC          | 11.28        | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) |

### Legacy Agreement MKR Distributions
```

**Authorization**: [SES](https://mips.makerdao.com/mips/details/MIP40c3SP17), [BA Labs](https://mips.makerdao.com/mips/details/MIP40c3SP25)

If this executive proposal passes, the following transfers will be made per Legacy Core Unit MIP agreements:

- [SES](https://mips.makerdao.com/mips/details/MIP40c3SP17) - 508.55 MKR - [0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6](https://etherscan.io/address/0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6). (Note - This figure covers the 278 MKR owed to SES from November plus the remainder of the legacy MKR agreements owed to the team, following their offboarding.)

> - [BA Labs](https://mips.makerdao.com/mips/details/MIP40c3SP25) - 175.00 MKR - [0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c](https://etherscan.io/address/0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c).
>   
> 
> ### Launch Project Funding

Reviewer: ```suggestion
- [BA Labs](https://mips.makerdao.com/mips/details/MIP40c3SP25) - 175.00 MKR - [0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c](https://etherscan.io/address/0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c).
  
### Launch Project Funding
```

- Transfer **4,500,000 DAI** to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).  
- Transfer **580.00 MKR** to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).

### End Token Streams

**Authorization**: [DECO](https://forum.makerdao.com/t/mip39c3-sp12-core-unit-offboarding-deco/22333/3), [SES](https://forum.makerdao.com/t/mip39c3-sp11-core-unit-offboarding-ses/22332/2)  
**Proposal**: [DECO](https://forum.makerdao.com/t/mip39c3-sp12-core-unit-offboarding-deco/22333), [SES](https://forum.makerdao.com/t/mip39c3-sp11-core-unit-offboarding-ses/22332)

If this executive proposal passes, the DAI [Token Streams](https://manual.makerdao.com/module-index/module-token-streaming#trade-offs) will be halted (`yank`) for [SES](https://makerburn.com/#/expenses/vesting/0xa4c22f0e25C6630B2017979AcF1f865e94695C4b/DAI/15) and [DECO](https://makerburn.com/#/expenses/vesting/0x2Cc583c0AaCDaC9e23CB601fDA8F1A0c56Cdcb71/DAI/21), following their respective offboarding.

> ### Aligned Delegate Buffer Payments

Reviewer: ```suggestion
### Derecognized Aligned Delegate Buffer Payments
```

> ### Aligned Delegate Buffer Payments

Reviewer: Match doc order to sheet order where possible so code and copy are aligned

**Authorization**: [Atlas Section 2.6.3.4.1](https://mips.makerdao.com/mips/details/MIP101#2-6-3-4-1)  
**Proposal**: [Forum Post](https://forum.makerdao.com/t/october-2023-aligned-delegate-compensation/22732#october-compensation-2)

> If this executive proposal passes, the following transfers will be made to pay out the remainder of the AD buffer following two Aligned Delegate Derecognitions:

Reviewer: ```suggestion
If this executive proposal passes, the following transfers will be made to pay out the remainder of the AD buffers following two Aligned Delegate derecognitions:
```

- Navigator - 20.84 MKR - [0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072)
- PALC - 6.95 MKR - [0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A](https://etherscan.io/address/0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A)

> ### RWA Agreement Upgrade

Reviewer: ```suggestion
### RWA Agreement Update
```

**Authorization**: [Forum Post](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/14)

> If this executive proposal passes, the HVBank (RWA009-A) [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter will be updated to to [QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT](https://ipfs.io/ipfs/QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT).
> 
> 
> ## Review

Reviewer: ```suggestion
If this executive proposal passes, the HVBank (RWA009-A) [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter will be updated to to [QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT](https://ipfs.io/ipfs/QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT).

## Review
```
